extensions/vertx-core/runtime/src/main/java/io/quarkus/vertx/core/runtime/VertxCoreRecorder.java

```diff
@@ -132,11 +132,14 @@ public static Vertx initialize(VertxConfiguration conf) {
         }
     }
 
-    private static VertxOptions convertToVertxOptions(VertxConfiguration conf) {
+    private static VertxOptions convertToVertxOptions(VertxConfiguration conf, boolean allowClustering) {
         VertxOptions options = new VertxOptions();
-        // Order matters, as the cluster options modifies the event bus options.
-        setEventBusOptions(conf, options);
-        initializeClusterOptions(conf, options);
+
+        if (allowClustering) {
+            // Order matters, as the cluster options modifies the event bus options.
+            setEventBusOptions(conf, options);
```

Comment:
Even if there is no clustering shouldn't we still set the event bus options?

Reply:
@cescoffier told me they were all related to the clustering.

@cescoffier are we sure it will always be the case and there won't be event bus options not-clustering related?

Reply:
Yes, all the event bus options are related to clustering (SSL, TCP options and so on)